modules/models.py

```diff
@@ -213,6 +214,26 @@ class DiscordMember(BaseModel):
     date_polychamps_invite_sent = DateField(default=None, null=True)
     boost_level = SmallIntegerField(default=None, null=True)
 
+    def as_json(self) -> dict[str, Any]:
```

Comment:
here too

Reply:
This is a feature of Python 3.9, see [PEP 585](https://www.python.org/dev/peps/pep-0585/). If compatibility with 3.7 and 3.8 is required, `from __future__ import annotations` can be used.